CHANGELOG.md

```diff
@@ -0,0 +1,330 @@
+# _fastlane_ CHANGELOG
+
+---
+
+When contributing to fastlane, don't forget to add an entry to this CHANGELOG.md file by copy/pasting and adapting this entry template:
+
+```
+* Describe your change here. End it with a full-stop and 2 spaces.  
```

Comment:
I'd love to have a Danger script for that. [There's a "changelog" Danger plugin](https://github.com/dblock/danger-changelog) doing such syntax checks, but afaik it's a bit too restrictive on the syntax right now. Would be worth a try though once we make Danger working again.